src/PrestaShopBundle/DataCollector/HookDataCollector.php

```diff
@@ -140,6 +140,23 @@ private function stringifyHookArguments(array &$hooksList)
                 }
             }
         }
+        // foreach ($hooksList as $hookList) {
+        //     foreach ($hookList as $hook) {
+        //         $hook['args'] = $this->cloneVar($hook['args']);
+
+        //         foreach ($hook['modules'] as $modulesByType) {
+        //             foreach ($modulesByType as $type => $module) {
+        //                 if (empty($module)) {
+        //                     unset($modulesByType[$type]);
```

Comment:
Commented code should not be submitted in PR 😉

Reply:
Sorry, until @boubkerbribri helped me, Iwas fighting with CI to understand what is wrong ^^'
I think this fix hasn't finished to torture me yet :sob: ...